config.yml

```diff
@@ -35,13 +35,15 @@ before:
   title: Create a staging workflow
   body: 01_label-trigger.md
   head: staging-workflow
+  data:
+    workflowUrl: '%payload.repository.html_url%/new/%payload.pull_request.head.ref%?filename=.github/CHANGETHIS/deploy-staging.yml'
```

Comment:
How can I test this?

Reply:
Hi @bdougie! Thank you for making these changes! 

There is a draft deployment automatically for PRs that will let us test. But, it looks like there was a problem with the deployment. I'll take a peek and see if I can get the build to succeed, then test.

Reply:
Hmm it seems that draft courses aren't created for PRs from forks. 🤔 I'll open an issue in Learning Lab about that. 

In the meantime, I'm opening #12 with your changes but based on a branch in this repo so it will hopefully build. 🤞

Reply:
For the sake of simplicity, and that I need to make a few more changes, I'm going to close this PR and continue working in #12. Thank you again @bdougie!